### PR TITLE
[docs] Update README.md

### PR DESCRIPTION
**Pull request by joshworkscivicactions**
Updated with communications procedures and refined responses.

## Description

See _issueid_. 

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: helpdesk-responses/README.md [hunk 1]
```diff
@@ -1,28 +1,526 @@
...
+| Training   | https://cms-training-pion6wlxie1t0mckcgwjl4kid1k1haqe.demo.cms.va.gov |
+| Tugboat    | https://tugboat.vfs.va.gov                                   |
+
+### **What are the credentials needed to login to the Production, Tugboat or Training servers?** 
+
+| Site       | Username          | Password      |
+| ---------- | ----------------- | ------------- |
+| Production | your PIV card     | your PIV card |
+| Training   | your VA.gov email | Drupal8       |
+| Tugboat    | your VA.gov eamil | Drupal8       |
```
Comment (swirtSJW) [inline helpdesk-responses/README.md:35]: The Tugboat dashboard uses github for access.

File: helpdesk-responses/README.md [hunk 2]
```diff
@@ -1,28 +1,526 @@
...
+
+## Defect or bug
+
+
+
+## Feature Request or Enhancement 
+
+# Communications with CMS Users
+
+####
```
Comment (swirtSJW) [inline helpdesk-responses/README.md:19]:
```suggestion

```

**Review by swirtSJW — COMMENTED** [review]
I am converting this to draft until we decide if this is really where we want canned answers to live.  https://dsva.slack.com/archives/CT4GZBM8F/p1625668316355000

**Comment by cmaeng** [comment]
This has been subsumed by Knowledge Base work. Closing to no further action.

**Pull request closed without merging**